data/lang/ui-core/de.json

```diff
@@ -715,6 +715,10 @@
     "description": "",
     "message": "Vollbildmodus"
   },
+  "GAME_OPTIONS": {
```

Comment:
You only need to update the en.json file, it's the "master", this will then propagate new strings to all other language files (might be temporarily not working though, but that's on me).

Reply:
As I speak German, I added the German translation as well..

Reply:
Your edit will probably get over-written by transifex. I can never quite remember what happens when editing the non-en.json files.

If you want to maintain the german translation, that should be done through transifex.

Reply:
Removed the german translations.. will add them again when they become available on transifex.

That said, I think it would be better if people could supply translations however they wished, with transifex being one option. transifex should populate itself from the existing data.. but no idea if that actually is possible. First time hearing about that system.